rpms/build.sh

```diff
@@ -51,11 +52,15 @@ $linux rpmbuild --define "_topdir ${rpm_build_dir}" --define "_threads ${threads
         --define "_debugenabled ${debug}" --target ${architecture_target} \
         -ba ${rpm_build_dir}/SPECS/${package_name}.spec
 
-rpm_file="wazuh-${build_target}-${wazuh_version}-${package_release}.${architecture_target}.rpm"
-pkg_path="${rpm_build_dir}/RPMS/${architecture_target}"
+rpm_file="wazuh-${build_target}-${wazuh_version}-${package_release}*"
+pkg_path="${rpm_build_dir}/RPMS"
 
 if [[ "${checksum}" == "yes" ]]; then
-    cd ${pkg_path} && sha512sum ${rpm_file} > /var/local/checksum/${rpm_file}.sha512
+    find ${pkg_path} -maxdepth 2 -type f -name "${rpm_file}" -exec sh -c 'sha512sum {} > {}.sha512 && mv {}.sha512 /var/local/checksum'  \;
 fi
 
-mv ${pkg_path}/${rpm_file} /var/local/wazuh
\ No newline at end of file
+if [ "${src}" == "yes" ]; then
```

Comment:
Use a single `=`

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/322/commits/1c0fc432f8a6a01bc925799249c0761244cb4f1f

Reply:
Check the syntax of the script. Where you are using `[[ expr ]]` you must use `==` when comparing variables, but if you use `[ expr ]` you must use `=`. In the commit 1c0fc43 you changed this. 

Can you review it again and changed only those lines where there's something like `[ x == y ]`?